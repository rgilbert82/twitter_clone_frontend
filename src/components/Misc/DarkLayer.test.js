import React from 'react';
import { shallow } from 'enzyme';
import { DarkLayer } from '.';

describe(DarkLayer, () => {
  let component;

  beforeEach(() => {
    component = <DarkLayer />;
  });

  it("renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
