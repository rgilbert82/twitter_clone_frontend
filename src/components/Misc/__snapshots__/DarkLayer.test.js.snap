// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class DarkLayer extends _react2.default.Component {
  constructor(props) {
    super(props);
    this.closeLayer = this.closeLayer.bind(this);
  }

  closeLayer(e) {
    e.preventDefault();
    this.props.closeModal();
  }

  render() {
    return (
      _react2.default.createElement("div", { onClick: this.closeLayer, className: "dark_layer", __source: { fileName: _jsxFileName, lineNumber: 16 }, __self: this },
        _react2.default.createElement("a", { href: "", className: "close_modal", __source: { fileName: _jsxFileName, lineNumber: 17 }, __self: this }, "close")));


  }} renders properly 1`] = `<DarkLayer />`;
